sbt-plugin/src/sbt-test/gen-java/01-gen-basic-java/src/main/protobuf/helloworld.proto

```diff
@@ -21,9 +23,11 @@ service GreeterService {
 // The request message containing the user's name.
 message HelloRequest {
     string name = 1;
+    google.protobuf.Timestamp timestamp = 2;
 }
 
 // The response message containing the greetings
 message HelloReply {
     string message = 1;
+    google.protobuf.Timestamp timestamp = 2;
```

Comment:
Either remove the timestamp here or (perhaps better) add the option to the `PB.protocOptions` in the `build.sbt` of this test

Reply:
I will add the options, as this will be able to serve as the documentation later.